examples/conftest.py

```diff
@@ -1,9 +1,15 @@
 from importlib.util import find_spec
 from packaging.version import Version
+import platform
 
-collect_ignore_glob = [
-    "tiling.ipynb",
-]
```

Comment:
Revert this, I don't think tiling is ready for testing. Can see it creates `examples/user_guide/s3:/` folder when building the documentation, which is not correct.